packages/schematics/angular/guard/files/__name@dasherize__.guard.ts.template

```diff
@@ -1,5 +1,5 @@
 import { Injectable } from '@angular/core';
-import { <%= implementationImports %>ActivatedRouteSnapshot, RouterStateSnapshot, UrlTree } from '@angular/router';
+import { <%= implementationImports %>, UrlTree } from '@angular/router';
```

Comment:
If `UrlTree` is always used shouldn't it be part of `routerSnapshotsNameImports`? so that it's also sorted?

Reply:
Yes it should. But can we find another name for the constant, because it's no longer just about snapshot imports? or names with snapshot as suffix?

Reply:
@alan-agius4  I just notice `routerSnapshotsNameImports` is only for other implementations than `canLoad`. `UrlTree` is already sorted so it does'nt matter to add this to the array before sort.

So two options:

- Keep UrlTree in the template, because it is already sorted and always comes last
- Or we add an extra array. e.g. routerCommonImports, which only contains a single UrlTree element and is added to the routerNamedImports before sorting 

Which one should we go for ?

Reply:
I think adding in the array is more future proof. 

See an example below;

```
 const commonRouterNameImports = ['ActivatedRouteSnapshot', 'RouterStateSnapshot', 'UrlTree'];
```

Reply:
done